### PR TITLE
Document how to build on macOS (stubbing out libostree support)

### LINKED ISSUE #380: Build failure on osx
As per the README

```
brew install gpgme
git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
```

Output:

```
go build -ldflags "-X main.gitCommit=91606d49f2f7d17c226940e98b5f42500a4c3efb" -gcflags "" -tags "btrfs_noversion libdm_no_deferred_remove" -o skopeo ./cmd/skopeo
# pkg-config --cflags ostree-1 ostree-1 ostree-1 ostree-1 ostree-1 ostree-1
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
pkg-config: exit status 1
make: *** [binary-local] Error 2
```

### PR DESCRIPTION
**Pull request by mtrmac**
Fixes #380
(minimally; autodetection would be better)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +123,8 @@ Make sure to clone this repository in your `GOPATH` - otherwise compilation fail
 
 ```sh
 $ git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
-$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
+Fedora$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
+macOS$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local BUILDTAGS=containers_image_ostree_stub
 ```
```
Comment (TomSweeneyRedHat) [inline README.md]: I didn't see anything in the installing or making documents sections for macOS$.  Can we add stuff there too?  Is it simply 'make install' on macOS?

**Comment by rhatdan** [comment]
How about 
cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local  BUILTAGS=$([ $(uname) == "Linux1" ]  && echo containers_image_ostree_stub)

File: README.md (hunk 1 above)
Comment (mtrmac) [inline README.md]: That depends on someone testing this and providing the content… Just in time, see #383 .

(Ideally, we would want the commands to be simple, and the same, everywhere. The manual `BUILDTAGS=` here is just an artefact of not atomating things enough.)

**Comment by mtrmac** [comment]
> How about
cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local BUILTAGS=$([ $(uname) == "Linux1" ] && echo containers_image_ostree_stub)

Yes, such a condition should exist, but it should exist inside the `Makefile`’s `binay-local` target, not in the end-user instructions. Hoping for a tested PR… which is now in #382.

**Comment by mtrmac** [comment]
Closing, #382 is a better solution.

**Pull request closed without merging**